.github/workflows/pull_request_label.yml

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2022 Sharezone UG (haftungsbeschränkt)
+# Licensed under the EUPL-1.2-or-later.
+#
+# You may obtain a copy of the Licence at:
+# https://joinup.ec.europa.eu/software/page/eupl
+#
+# SPDX-License-Identifier: EUPL-1.2
+
+name: pull-request-labeler
+
+on:
+  # Trigger "pull_request" would not work for pull request from a fork because
+  # the secrets would be not accessible. "pull_request_target" uses the context
+  # from the base branch which allows to use secrets for pull requests from a
+  # fork.
+  pull_request_target:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - closed
+
+# Declare default permissions as none.
+#
+# Having workflows without default permissions is considered a bad security
+# practice and it is causing alerts from our scanning tools.
```

Comment:
```suggestion
# Having workflows with default permissions is considered a bad security
# practice and it is causing alerts from our scanning tools.
```

Shouldn't it be this?

Reply:
It means the default permission in workflow. In this case

```yaml
permissions: {}
```